mpls-lab/library/conn_test_links.py

```diff
@@ -0,0 +1,54 @@
+# Krlosromero 20200321,test link connectivity
+import yaml
+import time
+from netmiko import Netmiko  
+from pprint import pprint
+from netaddr import IPNetwork
+
+# initial hosf location file --> it should be with args
+location = '../extra/hosts_file.conf'
+
+# reading yamls
+def doc_read(location):
+    with open(location, 'r') as f:
+        data = yaml.safe_load(f)
+        return(data)
+
+# get Peer ip on the link --> address has to be of type IPNetwork
+def get_peer_ip(address):
+    for host in address.iter_hosts():
+        if address.ip != host:
+            return host
+
+def conn_link_tests(data):
+    results = dict()
+    for host,mgmt_ip in data.items(): 
+        host_response = {}
+        # creating netmiko connector
+        net_connect = Netmiko(mgmt_ip, username="netops", password="netops123456",   device_type="cisco_ios",)
```

Comment:
Can you change the creds to be inserted through a prompt? like `getpass`?

Reply:
updated